smart-contracts/migrations/2_next.js

```diff
@@ -16,55 +16,48 @@ module.exports = function(deployer, network, accounts) {
   let operator;
   let owner;
   let initialValidators = [];
-  let initialPowers = [];
-  let consensusThreshold;
-
-  // Input validation for general env variables
-  if (process.env.CONSENSUS_THRESHOLD.length === 0) {
-    return console.error(
-      "Must provide consensus threshold as environment variable."
-    );
-  }
-  consensusThreshold = process.env.CONSENSUS_THRESHOLD;
+  let consensusThreshold = (process.env.CONSENSUS_THRESHOLD === 0 ? process.env.CONSENSUS_THRESHOLD:70)
+  let localValidatorCount = Number(process.env.VALIDATOR_COUNT === 0 ? process.env.VALIDATOR_COUNT:1)
```

Comment:
same comments for validator and power variable. I believe the ternary operator should be reversed so we don't have an incorrect configuration of 0 validators and 0 powers.